src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -1681,6 +1698,32 @@ logical function mixedlayer_restrat_init(Time, G, GV, US, param_file, diag, CS,
     call get_param(param_file, mdl, "USE_STANLEY_TVAR", CS%use_Stanley_ML, &
              "If true, turn on Stanley SGS T variance parameterization "// &
              "in ML restrat code.", default=.false.)
+    call get_param(param_file, mdl, "USE_CR_GRID", CS%Cr_grid, &
+             "If true, read in a spatially varying Cr field.", default=.false.)
+    call get_param(param_file, mdl, "USE_MLD_GRID", CS%MLD_grid, &
+             "If true, read in a spatially varying MLD_decaying_Tfilt field.", default=.false.)
+    if (CS%MLD_grid) then
+      call get_param(param_file, mdl, "MLD_TFILT_FILE", filename, &
+             "The path to the file containing the MLD_decaying_Tfilt fields.", &
+             default="")
+      call get_param(param_file, mdl, "MLD_TFILT_VAR", varname, &
+              "The variable name for MLD_decaying_Tfilt field.", &
+              default="MLD_tfilt")
+      filename = trim(inputdir) // "/" // trim(filename)
+      allocate(CS%MLD_filtered_space(G%isd:G%ied,G%jsd:G%jed), source=0.0)
+      call MOM_read_data(filename, varname, CS%MLD_filtered_space, G%domain, scale=US%s_to_T)
+    endif
+    allocate(CS%Cr_space(G%isd:G%ied,G%jsd:G%jed), source=CS%Cr)
+    if (CS%Cr_grid) then
+      call get_param(param_file, mdl, "CR_FILE", filename, &
+             "The path to the file containing the Cr fields.", &
+             default="")
+      call get_param(param_file, mdl, "CR_VAR", varname, &
+              "The variable name for Cr field.", &
+              default="Cr")
+      filename = trim(inputdir) // "/" // trim(filename)
+      call MOM_read_data(filename, varname, CS%Cr_space, G%domain, scale=US%s_to_T)
```

Comment:
CS%Cr_space is a nondimensional variable, so it is wrong to have the `scale=US%s_to_T` argument on this `read_data()` call.